src/main/java/com/gtnewhorizons/angelica/compat/mojang/NativeImage.java

```diff
@@ -61,15 +62,15 @@ public void downloadTexture(int level, boolean bl) {
 
 //        final int width = GL11.glGetTexLevelParameteri(GL11.GL_TEXTURE_2D, level, GL11.GL_TEXTURE_WIDTH);
 //        final int height = GL11.glGetTexLevelParameteri(GL11.GL_TEXTURE_2D, level, GL11.GL_TEXTURE_HEIGHT);
-        try (final MemoryStack stack = stackPush()) {
```

Comment:
pretty sure the modern implementation uses this, not sure why we'd want to remove it.